src/main/java/org/ays/common/model/AysAuditLog.java

```diff
@@ -0,0 +1,128 @@
+package org.ays.common.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.ays.common.model.request.AysHttpHeader;
+import org.ays.common.model.request.AysHttpServletRequest;
+import org.ays.common.model.response.AysHttpServletResponse;
+import org.ays.common.util.AysJsonUtil;
+import org.ays.common.util.AysRandomUtil;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents an audit log for tracking HTTP request and response details.
+ * This class captures metadata about the user request, the request itself,
+ * the server's response, and timestamps for auditing purposes.
+ */
+@Getter
+@Setter
+public class AysAuditLog {
```

Comment:
içe içe yapılar olduğu için mi @builder annotation'ı kullanmak yerine kendimiz yazdık basitçe ?

Reply:
@alpemreelmas `@Builder` anotasyonunda kendisi otomatik oluşturuyor ve özelleştirilemiyor. Ben burada özelleştirmek istediğim için kendim yazmak istedim. `@Builder` anotasyonuyla oluşturulan yapı ile birebir aynı olmuyor ondan dolayı yazmıştım 👍🏼